src/css/buttons.css

```diff
@@ -0,0 +1,36 @@
+.radio__container {
+    @apply block h-6 w-6 relative cursor-pointer;
+    border-radius: 50%;
```

Comment:
Why haven't you used the value rounded-full?!
If it's not in the config I think it should be added asap
and even if you won't use tailwind class 
50% as a value for a circle sometimes become distorted 
always use a high number of pixels for a perfect circle (ex:1000px )

Reply:
It wasn't there. I have added it and used it. Used higher pixel values instead of percentage.